Vagrantfile

```diff
@@ -0,0 +1,61 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# This script provisions a vagrant development environment with local
+# postgres and redis servers for development and testing.
+PROVISION = <<SCRIPT
+set -e
+
+# cd to /vagrant on login
+cd /vagrant
+grep -q 'cd /vagrant' ~/.bashrc || echo 'cd /vagrant' >> ~/.bashrc
+
+# Install system packages
+sudo apt-get update --quiet
+sudo apt-get install -y $(cat debian_packages.lst) postgresql
+
+# Upgrade pip
+sudo pip3 install --upgrade pip
```

Comment:
@omarkhan It's tempting to use global packages given that the application is isolated on the VM, but that also means that if we do need to install a second app on the VM later on, we'll have to migrate to virtualenvs then. I'd rather apply good practices from the start, and use a venv. the effort is minimal and the commands are already listed in the README.

That has also the advantage of not having to handle the differences between people using a virtualenv or not, or having to redefine the location of the `src` dir, right below.